package.json

```diff
@@ -53,14 +53,14 @@
   "homepage": "https://github.com/PocketDerm/radiance-ui#readme",
   "devDependencies": {
     "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
+    "@babel/core": "^7.11.1",
     "@babel/plugin-proposal-class-properties": "^7.1.0",
```

Comment:
We should bump all the babel packages in lock-step: @babel/cli, @babel/plugin-proposal-class-properties, @babel/plugin-proposal-export-default-from, @babel/plugin-proposal-export-namespace-from, @babel/plugin-transform-parameters, @babel/preset-react, @babel/register

Reply:
piushing this soon

Reply:
pushed all updated dependecies. Bundle size matches what we have in curology now